services/net/contentmigration/ContentMigrationAction.cs

```diff
@@ -130,19 +131,20 @@ public ContentMigrationAction(MigrationSourceContext sourceContext, ContentMigra
             {
                 var baseFilter = contentMigrator.GetBaseFilter();
                 IQueryable<NewsItem> items = GetFilteredNewsItems(_sourceContext.NewsItems, baseFilter, skip, count, manager.Ingest.LastRanOn, importDateStart, importDateEnd, creationDateOfLastImport);
-                var lastNewsItem = items.Last();
                 count = items.Count();
+                if (count == 0) break;
```

Comment:
Why bother with this?  The foreach on 139 will exit immediately if there are no items.